Fix parallel make failure

Upstream-Status: Submitted [Maintainer directly]

Add a missing dependency which resulted in a race leading to failure
on larger values of -j.

Signed-off-by: Darren Hart <dvhart@linux.intel.com>

Index: gnu-efi-3.0/Makefile
===================================================================
--- gnu-efi-3.0.orig/Makefile
+++ gnu-efi-3.0/Makefile
@@ -42,6 +42,8 @@ include $(SRCDIR)/Make.defaults
 
 SUBDIRS = lib gnuefi inc apps
 
+gnuefi: lib
+
 all:	check_gcc $(SUBDIRS)
 
 $(SUBDIRS):
